### PR TITLE
fix: include npm audit reference link

### PR DESCRIPTION
**Pull request by RafaelGSS**
as titled

### CODE REVIEW

File: apps/site/pages/en/learn/getting-started/security-best-practices.md [hunk 1]
```diff
@@ -449,6 +449,7 @@ The [OpenSSF][] is leading several initiatives that can be very useful, especial
 [typosquatting]: https://en.wikipedia.org/wiki/Typosquatting
 [Mitigations for lockfile poisoning]: https://blog.ulisesgascon.com/lockfile-posioned
 [`npm ci`]: https://docs.npmjs.com/cli/v8/commands/npm-ci
+[`npm audit`]: https://docs.npmjs.com/cli/v9/commands/npm-audit
```
Comment (mikeesto) [inline apps/site/pages/en/learn/getting-started/security-best-practices.md]:
```suggestion
[`npm-audit`]: https://docs.npmjs.com/cli/v9/commands/npm-audit
```

**Review by mikeesto — APPROVED** [review]
LGTM, thanks

File: apps/site/pages/en/learn/getting-started/security-best-practices.md [hunk 2]
```diff
@@ -449,6 +449,7 @@ The [OpenSSF][] is leading several initiatives that can be very useful, especial
 [typosquatting]: https://en.wikipedia.org/wiki/Typosquatting
 [Mitigations for lockfile poisoning]: https://blog.ulisesgascon.com/lockfile-posioned
 [`npm ci`]: https://docs.npmjs.com/cli/v8/commands/npm-ci
+[`npm-audit`]: https://docs.npmjs.com/cli/v9/commands/npm-audit
```
Comment (bmuenzenmeyer) [inline apps/site/pages/en/learn/getting-started/security-best-practices.md]: unless the version is important (the text suggests not), removing the version will default it to latest, but leaving it there will bring the visitor to outdated data.

if people approve this suggestion, we should comb through all the docs for this pattern (see the above line, elsewhere too)

```suggestion
[`npm-audit`]: https://docs.npmjs.com/cli/commands/npm-audit
```

**Review by AugustinMauroy — APPROVED** [review]
LGTM ! With Brian's comment

File: apps/site/pages/en/learn/getting-started/security-best-practices.md (hunk 2 above)
Comment (bmuenzenmeyer) [reply apps/site/pages/en/learn/getting-started/security-best-practices.md]: fixed these references via https://github.com/nodejs/nodejs.org/commit/39855a75a4b382d0fbc39c1f3b611860b91b12f7 as part of troubleshooting something else

**Review by bmuenzenmeyer — APPROVED** [review]

**Pull request merged**